src/main/java/org/opentripplanner/client/parameters/TripPlanParametersBuilder.java

```diff
@@ -48,6 +49,11 @@ public TripPlanParametersBuilder withSearchDirection(SearchDirection searchDirec
     return this;
   }
 
+  public TripPlanParametersBuilder withSearchWindow(Long searchWindow) {
```

Comment:
The search window should be of type `java.time.Duration`.

Reply:
And we get the value for the actual query by `searchWindow.toMinutes()`?